express-exercise.js

```diff
@@ -0,0 +1,47 @@
+//Pair programming activity - Mini REST API exercise: Robyn and Zarina
+
+const express = require("express");
+const app = express();
+const port = 3000;
+
+app.get("/", (req, res) => {
+    res.sendFile(__dirname + `/index.html`)});
+
+let list = [
+    {name: "staff", member: "michelle@techtonica.org"},
+    {name: "staff", member: "talea@techtonica.org"},
+    {name: "apprentice", member: "robyn@techtonica.org"},
+    {name: "apprentice", member: "zarina@techtonica.org"},
+    {name: "apprentice", member: "ariel@techtonica.org"},
+    {name: "apprentice", member: "zhag@techtonica.org"}
+];
+
+app.route("/list").get((req, res) => {
+    res.status(200).send(list);
```

Comment:
the assignment was to _only_ respond with a list of names, so you'd need to collect the unique names from your `list` object